site/content/docs/5.0/examples/carousel-rtl/index.html

```diff
@@ -59,7 +59,7 @@ <h1>عنوان المثال.</h1>
           <div class="carousel-caption">
             <h1>عنوان مثال آخر.</h1>
             <p>ثم تزامناً الفرنسي الإقتصادية دار. لكل عن الضغوط المتّبعة, أن حتى إختار المدن بالإنزال. عن الشمل بالفشل تلك, بل أراض أوزار بلديهما حول. دون لكون والتي ثم, كُلفة ويعزى استطاعوا أن لمّ. جُل بخطوط واحدة البشريةً.</p>
-            <p><a class="btn btn-lg btn-primary" href="#" role="button">أعرف أكثر</a></p>
```

Comment:
isn't the role needed since we use `a`?

Reply:
they're not buttons though, are they? they're links going to some page, no?

Reply:
Ah, true, those are captions. Might be worth splitting these to a separate PR?

Reply:
to clarify: if a link acts as a link (goes to another page or whatever), rather than doing something dynamic (opening a modal dialog, a dropdown, etc), then it should *not* have `role="button"`. and for these examples, I'm assuming these links are meant to signify links that act as links. whether the link is visually styled as a `btn` is usually irrelevant ("call-to-action links"), what counts is what the purpose of the control is.

Reply:
> Might be worth splitting these to a separate PR?

admittedly, i'm already getting lost with the various PRs i had going at the moment, and as this is just tweaking something in an example, in passing, not sure if it's really useful to split? your call though.

Reply:
ok reverted this change with the latest commit

Reply:
Thanks! Don't forget to submit a separate PR please

Reply:
done https://github.com/twbs/bootstrap/pull/32788